composer.json

```diff
@@ -13,7 +13,7 @@
     ],
     "require": {
         "php": ">=5.3.2",
-        "guzzle/guzzle": "~3.0",
+        "guzzlehttp/guzzle": "~5.2",
```

Comment:
As far as I'm aware 5.0 should work, I only used 5.2 as it was the latest. As you state, it makes sense to use ~5.0 and support the maximum amount of people. Hopefully Guzzle follow [semver](http://semver.org/) in which case we shouldn't be dependent on anything 5.2 specific in the API since we migrated from 3.